modules/sharethroughBidAdapter.js

```diff
@@ -261,20 +328,28 @@ export const sharethroughAdapterSpec = {
   },
 
   getUserSyncs: (syncOptions, serverResponses) => {
```

Comment:
Since we finally don't change the current user sync I think we don't need to touch this method.